dm/pkg/schema/tracker_test.go

```diff
@@ -114,6 +116,8 @@ func (s *trackerSuite) TestTiDBAndSessionCfg(c *C) {
 	c.Assert(mock.ExpectationsWereMet(), IsNil)
 	err = tracker.Exec(context.Background(), "", "create database testdb;")
 	c.Assert(err, IsNil)
+	err = tracker.Close()
+	c.Assert(err, IsNil)
```

Comment:
maybe make this and below two into a `defer` after `newTracker` too

Reply:
`tracker` will be lost by overwritten when defer closure call, I need to assign every tracker for every `NewTracker` then

Reply:
just in case `tracker.close` is not called for previous assert may failed